tests/actions/ReportTest.ts

```diff
@@ -1186,14 +1201,15 @@ describe('actions/Report', () => {
                 key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${REPORT_ID}`,
                 callback: (reportActions) => {
                     Onyx.disconnect(connection);
-                    expect(reportActions?.[reportActionID]).not.toBeNull();
-                    expect(reportActions?.[reportActionID].reportActionID).toBe(reportActionID);
+                    const reportAction = reportActionID ? reportActions?.[reportActionID] : null;
+                    expect(reportAction).not.toBeNull();
+                    expect(reportAction?.reportActionID).toBe(reportActionID);
```

Comment:
What's the change here?

Reply:
The value of `reportActions?.[reportActionID]` is stored in `const reportAction`.

So, instead of `reportActions?.[reportActionID].reportActionID`, we use `reportAction?.reportActionID`